internal/io/sql/reader.go

```diff
@@ -70,13 +73,13 @@ func ReadSQL(rows *sql.Rows) (map[string]types.DataSlice, error) {
 	var columns []*Column
 	for rows.Next() {
 		if err := rows.Err(); err != nil {
-			return nil, err
+			return nil, errors.New("ReadSQL Rows", err.Error())
 		}
 		// Allocate columns for the returning query
 		if columns == nil {
 			cols, err := rows.Columns()
 			if err != nil {
-				return nil, err
+				return nil, errors.New("ReadSQL Columns", err.Error())
```

Comment:
Picky: Consider using `errors.Propagate` here to avoid throwing away the original error.
This is a general comment.